core/templates/components/graph-services/graph-layout.service.spec.ts

```diff
@@ -1201,7 +1227,13 @@ describe('Graph Layout Service', () => {
     let modifiedNodeData = sgls.modifyPositionValues(
       nodeData, graphWidth, graphHeight);
 
-    expect(modifiedNodeData).not.toEqual(nodeData1);
-    expect(modifiedNodeData).toEqual(nodeData);
+    for (let nodeId in nodeData1) {
```

Comment:
This is still too big to understand. Better to do just one or two nodes and add some explanation for why the modified values are what they are so that the reader can verify. (The current test doesn't provide much insight, it just says "this blob of stuff is the correct thing" but doesn't make it easier for the reader to understand why it's the correct thing.)

Reply:
Added an explanation

Reply:
Don't see it.

Reply:
Sorry - I forgot to push the changes. Updated now